lambdatimes/src/components/Content/Tab.js

```diff
@@ -18,5 +20,7 @@ const Tab = props => {
 };
 
 // Make sure you include PropTypes on your props.
-
+Tab.PropTypes = {
```

Comment:
If you are using proptypes, be sure to import the package at the top of your file!